docs/email/templates/code/promotional.html

```diff
@@ -326,62 +288,46 @@
-                                                </td>
-                                            </tr>
-                                        </table>
-                                    </div>
-                                    <!--[if mso]>
+                        <div style="display:inline-block; margin: 0 -2px; width:100%; min-width:200px; max-width:340px; vertical-align: top;" class="stack-column">
+                            <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+                                <tr>
+                                    <td style="padding: 30px 30px 15px; text-align: left; font-family: arial, sans-serif; color: #ffffff;" class="sm-px sm-pt">
+                                        <h1 style="margin: 0 0 15px; font-weight: bold; font-size: 27px; line-height: 27px;">We’re announcing something</h1>
+                                        <p style="margin: 10px 0; font-size: 17px; line-height: 21px;">The excerpt or slogan of the email announcement can go here.</p>
                                     </td>
-                                    <td align="left" valign="top" width="340">
-                                    <![endif]-->
-                                    <div style="display:inline-block; margin: 0 -2px; width:100%; min-width:200px; max-width:340px; vertical-align: top;" class="stack-column-center">
-                                        <table role="presentation" cellspacing="0" cellpadding="0" border="0" width="100%">
+                                </tr>
+                                <tr>
+                                    <td style="padding: 0 30px 30px;" class="sm-px sm-pb">
+                                        <!-- Button : BEGIN -->
+                                        <table align="left" border="0" cellpadding="0" cellspacing="0" role="presentation">
                                             <tr>
-                                                <td style="padding: 30px 10px; font-family: arial, sans-serif; font-size: 15px; line-height: 21px; color: #ffffff; text-align: center;" class="sm-py">
-                                                    <img src="illo-scene-1.png" width="340" height="" alt="alt_text" border="0" style="display: block; margin: auto; width: 100%; max-width: 340px; font-family: arial, sans-serif; font-size: 15px; line-height: 16px; color: #ffffff;" class="g-img">
+                                                <td class="s-btn s-btn__white" style="border-radius: 4px; background: #ffffff;">
+                                                    <a class="s-btn s-btn__white" href="#" style="background: #ffffff; border: 1px solid #ffffff; box-shadow: 0 0 0 0 rgba(0,149,255,0); font-family: arial, sans-serif; font-size: 17px; line-height: 17px; color: #0077cc; text-align: center; text-decoration: none; padding: 13px 17px; display: block; border-radius: 4px; white-space: nowrap;">Medium White Button</a>
```

Comment:
Defininig color on a `box-shadow` that doesn't have any size seems unnecessary, right? Also, wouldn't `box-shadow: none;` work instead?

Reply:
You're right, we could probably remove this line entirely since there's no cascade we have to defend against in email. Wondering if this is a relic of me pasting something from our web design system.